tests/test_graphical_units/test_transform.py

```diff
@@ -132,6 +132,15 @@ def construct(self):
         self.play(FadeOut(square))
 
 
+class MatchPointsScene(Scene):
+    def construct(self):
+        circ = Circle(fill_color=RED, fill_opacity=0.8)
+        square = Square(fill_color=BLUE, fill_opacity=0.2)
+        self.add(circ)
+        circ.match_points(square)
+        self.wait()
```

Comment:
Just to be consistent with other tests
```suggestion
        self.play(circ.animate.match_points(square))
```